File: Lab2/Lab2.cpp
#include <iostream>

using namespace std;

double func(double x, double a) {
	double y;

	y = (x < 0) ? y = x : y = a;

	return y;
}

int main()
{
	double x, a, y;

	//input data
	cout << "x = "; cin >> x;
	cout << "A = "; cin >> a;

	y = func(x, a);

	printf("y = %f\n", y);

	return 0;
}

File: lab_2/lab_2.py

def func(x, a):
    if x < 0:
        y = x
    else:
        y = a
    return y

def main():
    try:
        x = float(input("x = "))
        a = float(input("A = "))
    except:
        print("Invalid data")
        return

    y = func(x, a)

    print(f"y = {y}")
    
main()

